gnmi/gnmit/gnmit.go

```diff
@@ -181,6 +181,27 @@ func NewServer(ctx context.Context, hostname string, sendMeta bool, tasks []Task
 	return c, gnmiserver, nil
 }
 
+func StartDatastoreServer(gnmiServer *GNMIServer) (func(), error) {
```

Comment:
add comment (especially clarifying return values

Reply:
oops I put the comment in the subsequent PR.